azure-stack/hci/manage/manage-syslog-forwarding.md

```diff
@@ -0,0 +1,469 @@
+---
+title: Manage syslog forwarding for Azure Stack HCI (preview)
+description: Learn about security features for Azure Stack HCI security information and event management (SIEM) (preview).
+author: alkohli
+ms.author: alkohli
+ms.topic: how-to
+ms.service: azure-stack
+ms.subservice: azure-stack-hci
+ms.date: 12/01/2023
+---
+
+# Manage syslog forwarding for Azure Stack HCI (preview)
+
+[!INCLUDE [hci-applies-to-23h2](../../includes/hci-applies-to-23h2.md)]
+
+This article describes how to manage components in the security layer of the Azure Stack HCI, version 23H2 solution for security information and event management (SIEM).
+
+For additional security considerations, see:
+
+- [Security baseline settings on Azure Stack HCI (preview)](/azure-stack/hci/concepts/secure-baseline).
+- [BitLocker encryption on Azure Stack HCI (preview)](/azure-stack/hci/concepts/security-bitlocker).
+
```

Comment:
The above two links need to be replaced by the corresponding conceptual articles that we will create. 
1. Security features - WDAC, security baseline, BitLocker encryption
2. Other security features - Local builtin user accounts, SIEM/syslog, secrets management

Reply:
Done > although the first link may eventually be to an article TBD that we have not yet developed.

Reply:
Link to single article

Reply:
Done > updated link.